engine/class_modules/sc_warrior.cpp

```diff
@@ -5215,18 +5224,19 @@ void warrior_t::apl_prot()
     prot->add_action( "potion,if=target.time_to_die<25" );
   }
 
-  prot->add_action( this, "Battle Cry", "if=cooldown.shield_slam.remains=0" );
-  prot->add_action( this, "Avatar" );
+  prot->add_action( this, "Avatar", "if=cooldown.demoralizing_shout.remains>5" );
   prot->add_action( this, "Demoralizing Shout" );
   prot->add_action( this, "Ravager", "if=talent.ravager.enabled" );
```

Comment:
Same as below. Although this is most likely an older issue that nobody noticed because ravager has been historically bad for prot.
```suggestion
  prot->add_talent( this, "Ravager" );
```